src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/dto/ServiceProviderDTO.kt

```diff
@@ -0,0 +1,13 @@
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.dto
+
+import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity.ServiceProvider
+
+class ServiceProviderDTO(
+  val name: String,
+) {
+  companion object {
+    fun from(serviceProvider: ServiceProvider): ServiceProviderDTO {
+      return ServiceProviderDTO(name = serviceProvider.name)
+    }
+  }
```

Comment:
Do we need a DTO just for a String ?

Reply:
good point. this matches with the UI expectations of the response though. and if we add more more SP fields in the future this existing one won't have to change